samtranslator/open_api/open_api.py

```diff
@@ -376,6 +379,75 @@ def add_tags(self, tags):
                 tag = {"name": name, self._X_APIGW_TAG_VALUE: value}
                 self.tags.append(tag)
 
+    def add_cors(
+        self,
+        allowed_origins,
+        allowed_headers=None,
+        allowed_methods=None,
+        expose_headers=None,
+        max_age=None,
+        allowed_credentials=None,
```

Comment:
why not `allow_credentials`?

Reply:
you are right. I will name all the properties consistently. I will modify `allowed` to `allow`